vertx-quickstart/src/main/java/org/acme/VertxResource.java

```diff
@@ -30,7 +32,7 @@
     @Path("/lorem")
     public Uni<String> readShortFile() {
         return vertx.fileSystem().readFile("lorem.txt")
-                .onItem().transform(content -> content.toString("UTF-8"));
+                .onItem().transform(content -> content.toString(StandardCharsets.UTF_8));
```

Comment:
This one might require a change in the guide.

Reply:
Ah good catch, will provide a PR there too